### PR TITLE
release: jdk19.0.2+7

### PR DESCRIPTION
**Pull request by steelhead31**
shipped:
x64 linux
aarch64 linux
ppc64le
arm32
s390x
alpine

### CODE REVIEW

**Review by gdams — APPROVED** [review]

File: linux/jdk/redhat/src/main/packaging/temurin/19/temurin-19-jdk.spec [hunk 1]
```diff
@@ -1,10 +1,10 @@
-%global upstream_version 19.0.1+10
+%global upstream_version 19.0.2+7
 # Only [A-Za-z0-9.] allowed in version:
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
 # also not very intuitive:
 #  $ rpmdev-vercmp 19.0.0.0.0___19.0.0.0.0+36
 #  19.0.0.0.0___36 == 19.0.0.0.0+36
-%global spec_version 19.0.1.0.0.10
+%global spec_version 19.0.2.0.0.7
 %global spec_release 2
```
Comment (zdtsw) [inline linux/jdk/redhat/src/main/packaging/temurin/19/temurin-19-jdk.spec:8]: these are still needed to set to 1 for suse and rhel

Comment (steelhead31) [reply linux/jdk/redhat/src/main/packaging/temurin/19/temurin-19-jdk.spec]: done!

**Review by zdtsw — APPROVED** [review]

**Pull request merged**